src/UWP/ArcGISRuntime.UWP.Viewer/Samples/Layers/ShowPopup/ShowPopup.xaml

```diff
@@ -0,0 +1,14 @@
+﻿<UserControl
+    x:Class="ArcGISRuntime.UWP.Samples.ShowPopup.ShowPopup"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:esri="using:Esri.ArcGISRuntime.UI.Controls"
+    xmlns:toolkit="using:Esri.ArcGISRuntime.Toolkit.UI.Controls">
+    <Grid>
+        <esri:MapView x:Name="MyMapView" GeoViewTapped="MapViewTapped" />
+        <Border Style="{StaticResource BorderStyle}">
+            <toolkit:PopupViewer x:Name="MyPopupViewer" Width="400" />
```

Comment:
I recommend adding a label telling the user to tap to identify; the border doesn't look great starting out empty - applies to multiple platforms

Reply:
I also recommend updating the UWP screenshot to show the sample with a popup selected